### ISSUE TITLE
Kubernetes Dashboard shows "???" over the places

### ISSUE THREAD

**Issue by mumoshu** [kubernetes/dashboard#1025]
#### Issue details

Kubernetes Dashboard shows "???" over the places.

(I'm replicating the [according issue in the minikube repo](https://github.com/kubernetes/minikube/issues/326) here because I'm not sure where is this issue coming from or have no idea how to drill down to the root cause)

##### Environment

minikube v0.5.0

```
Dashboard version: v1.1.0
Kubernetes version: v1.3.0
Operating system: boot2docker running on minikubeVM(VirtualBox)
```

##### Steps to reproduce

With minikube v0.5.0 installed:

```
$ minikube start
$ minikube dashboard
```

##### Observed result

As the title says, all the texts other than kubernetes objects' names are shown as '?????'

![image](https://cloud.githubusercontent.com/assets/22009/16828112/e6527918-49c7-11e6-80f5-b05e0d8bebe0.png)

##### Expected result

There are meaningful texts.

**Comment by bryk** [comment]
Hah, so this is funny. From you github profile I assume that you are from Japan. Our UI is translated into Japanese and when it detects that your default language is JA, then it switches to Japanese strings.

You can quickly fix this by changing your preferred browser lang to English. 

The question is why you have `??` instead of Japan characters. Do you maybe miss some fonts? Do you usually see Japanese characters on other websites?

**Comment by bryk** [comment]
cc @taimir

**Comment by mumoshu** [comment]
@bryk Thanks for taking your time.
Yes, I'm from Japan and switching preferred lang on my browser to English did fix the issue, like this:

![image](https://cloud.githubusercontent.com/assets/22009/16831043/b83bfbf6-49dd-11e6-88bd-1b13558e3edf.png)

I guess it isn't a fonts related issue because:

* I usually see Japanese characters without any problem on other websites.
* looking into HTML sources also shows many `??`s:

![image](https://cloud.githubusercontent.com/assets/22009/16831137/35dd3e44-49de-11e6-9bf2-0c9e9e83e72a.png)

**Comment by taimir** [comment]
OK, I'm on it. Let's see what causes this, `serve:prod` definitely displays the japanese text correctly so I suspect it might be a packaging issue.

**Comment by bryk** [comment]
@taimir Can you run this via minikube and see what's happened

**Comment by taimir** [comment]
Mhm, I'll try it out.

**Comment by taimir** [comment]
@bryk
Hey, sorry for the slight delay - I wanted to double check everything.

So it appears the problem is only in the uploaded dashboard image: `gcr.io/google_containers/kubernetes-dashboard-amd64:v1.1.0`. I tried using this one both in `minikube` and also manually starting it and attaching it to another cluster (with `docker run`). In both cases `???` is displayed instead of the Japanese translations.

But then I built a dashboard image myself using `gulp docker-image:release`, based on the current master branch. I uploaded it to my private DockerHub account: `docker.io/taimir93/dashboard-amd64:v1.1.0`. I then used this new image in minikube (updating the `minikube/deploy/addons/dashboard-rc.yaml` and rebuilding minikube) and it all worked out fine.

To be extra sure I also tried using `debian:jessie` as the BASE for dashboard in a third image, then I logged in in it to check the contents of the Japanese-localized `app.js` code. It was all as it should be.

Long story short: it appears that there is something wrong with how the `gcr.io/google_containers/kubernetes-dashboard-amd64:v1.1.0` was built and pushed. Could it be that the machine that was used for that has some issues with the Japanese symbols? It all works out fine on my Ubuntu 14.04 ...

**Comment by taimir** [comment]
@mumoshu In case you feel like experimenting, you could try using the image I just created:
`docker.io/taimir93/dashboard-amd64:v1.1.0`, then updating the contents of `minikube/deploy/addons/dashboard-rc.yaml` and rebuilding minikube with `make`.

You'd probably need to clean up the old minikube before that: `./out/minikube delete`, `make clean` and then `make`, `./out/minikube start`, `./out/minikube/dashboard` after you've updated the image in the `.yaml` file :)

**Comment by bryk** [comment]
Hmmm... I'll try to rebuild the image and see whether this changes something.

**Comment by bryk** [comment]
Ok, so this is fixed now. Try to rerun minikube or re-pull the image. If problem persists, remove the old version of the image from your machine.

The problem was bad build. We built the image on a machine with no locale support. This will not happen again.

**Comment by mumoshu** [comment]
@taimir @bryk I have just confirmed it working like a charm after pulling the image and recreating the kubernetes dashboard container. Thanks a lot!
